Gruntfile.js

```diff
@@ -25,8 +25,12 @@ module.exports = function(grunt) {
         babel: {
             options: {
                 sourceMap: true,
-                modules: 'amd',
-                blacklist: 'useStrict'
+                presets: [
+                    ['es2015', { modules: 'amd' }]
+                ],
+                plugins: [
+                    'add-module-exports'
```

Comment:
This plugin is needed to maintain the old AMD export behavior. By default, babel exports things into a default key now, like `{ default: Nugget, ... }`. This allows us to export just the `Nugget` object like we've done in the past.